pages/class/[classid].tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useContext } from "react";
```

Comment:
change the name of the file to [classId].tsx to be consistent with the query params